Unhide lv3:enter_switch option, it can be useful if a pc105 model
is selected with an Apple laptop.

Not submitted upstream.

Index: xkb-data/rules/base.o_s.part
===================================================================
--- xkb-data.orig/rules/base.o_s.part	2007-09-14 19:17:28.000000000 +0200
+++ xkb-data/rules/base.o_s.part	2007-09-14 19:17:46.000000000 +0200
@@ -42,6 +42,7 @@
   lv3:win_switch	=	+level3(win_switch)
   lv3:lwin_switch	=	+level3(lwin_switch)
   lv3:rwin_switch	=	+level3(rwin_switch)
+  lv3:enter_switch	=	+level3(enter_switch)
   caps:capslock		=	+capslock(capslock)
   caps:shiftlock	=	+capslock(shiftlock)
   caps:grouplock	=	+capslock(grouplock)
Index: xkb-data/symbols/level3
===================================================================
--- xkb-data.orig/symbols/level3	2007-09-14 19:16:52.000000000 +0200
+++ xkb-data/symbols/level3	2007-09-14 19:17:46.000000000 +0200
@@ -135,7 +135,7 @@
 // using the level3(enter_switch) map, the Enter key on the keypad
 // temporarily chooses the third shift level.  This is especially
 // useful for Mac laptops which miss the right Alt key.
-partial hidden modifier_keys
+partial modifier_keys
 xkb_symbols "enter_switch" {
   key <KPEN> {
     type[Group1]="ONE_LEVEL",
Index: xkb-data/rules/base.xml.in
===================================================================
--- xkb-data.orig/rules/base.xml.in	2007-09-14 19:17:28.000000000 +0200
+++ xkb-data/rules/base.xml.in	2007-09-14 19:17:46.000000000 +0200
@@ -3535,6 +3535,12 @@
           <_description>Right Alt key never chooses 3rd level.</_description>
         </configItem>
       </option>
+      <option>
+        <configItem>
+          <name>lv3:enter_switch</name>
+          <_description>Press Enter key on keypad to choose 3rd level.</_description>
+        </configItem>
+      </option>
     </group>
     <group allowMultipleSelection="false">
       <!-- Tweaking the position of the "Ctrl" key -->
